### PR TITLE
Adding Pull Request Landmines

### PR DESCRIPTION
**Pull request by tylermurry**
💣 **Pull Request Landmines for GitHub**
---
A simple, language agnostic solution for creating manual mutation tests directly in-line with a GitHub pull request

[Marketplace](https://github.com/marketplace/actions/pr-landmine) | [Repository](https://github.com/tylermurry/github-pr-landmine)

asdf

---
🚨 Note: This PR also resolves the linting issue by removing the least-impactful duplicate GitHub action link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 - [Official Site](https://github.com/features/actions)
 - [Official Documentation](https://help.github.com/en/actions)
-- [Official Actions organization](https://github.com/actions)
```
Comment (sdras) [inline README.md:42]: I don't want to remove this, it's an important link. The Other two are ok as additions in subcategories.

Comment (tylermurry) [reply README.md]: Reverted!

**Comment by tylermurry** [comment]
@sdras - Changes have been made but the build is currently failing due to linting errors outside of this PR. Is this still ok to merge through or would you like me to address them?

**Comment by sdras** [comment]
Looks great, thanks! Linting error was fixed in another PR

**Pull request merged**